docs/README_ja.md

```
@@ -102,4 +103,4 @@ MIT License
 [badge-ios]: http://img.shields.io/badge/-ios-CDCDCD.svg
 [badge-mac]: http://img.shields.io/badge/-macos-111111.svg
 [badge-watchos]: http://img.shields.io/badge/-watchos-C0C0C0.svg
-[badge-tvos]: http://img.shields.io/badge/-tvos-808080.svg
\ No newline at end of file
+[badge-tvos]: http://img.shields.io/badge/-tvos-808080.svg
```

markdownlint-cli2 (0.18.1)

Link and image reference definitions should be needed
Unused link or image reference definition: "badge-tvos"

(MD053, link-image-reference-definitions)